spec/controllers/hyrax/downloads_controller_spec.rb

```diff
@@ -56,7 +56,7 @@
           end
 
           it 'sends requested file content' do
-            get :show, params: { id: file_set, file: 'thumbnail' }
+            get :show, params: { id: file_set.id, file: 'thumbnail' }
```

Comment:
Why are we adding `.id` here? Is that necessary?  It should be fine to just say `id: file_set`

Reply:
Tests break if we don't have it.  It seems like we needed it for other controller tests too.